### PR TITLE
Toggle Button for Light/Dark added

### PR DESCRIPTION
**Pull request by Prashant-2024**
Issue No. #54 
Toggle Button added for Light and Dark mode of application.

### CODE REVIEW

**Comment by abhijeetnishal** [comment]
Hey @Prashant-2024,  there are couple of issue in this PR:
1. Component should be in TypeScript not JS and file should be in .tsx format for component.
2. In the current project, we already using "next-themes" so we can implement Light/dark mode feature directly using this. 
3. The switch between light and dark mode is just button not a toggle one, use toggle button.
4. The toggle button is present in Navbar not in page so that users can switch in any component
5. The button should be visible, currently it's not visible size is small as it's not a toggle button.

Try to understand the project first before contributing so that you can easily understand.

**Comment by Prashant-2024** [comment]
I'm sorry for causing this inconvenience, I missed the part where I had to implement it with typescript, it's still new to me. 
I will the correct the implementation ASAP, and share it with you.

**Comment by abhijeetnishal** [comment]
No problem @Prashant-2024, open source and gssoc is for learning.

**Comment by abhijeetnishal** [comment]
Hey @Prashant-2024, are you done with the changes?

**Comment by Prashant-2024** [comment]
Yes, I am close to it.
I will share them by today evening.
I know i have taken long time and causing delay, for this changes. I am currently travelling and don't have access to laptop.
I will share them ASAP

**Comment by Prashant-2024** [comment]
Below are the changes:

Dark Mode:

image

Light Mode:

image

@abhijeetnishal Request to look into it and let me know

File: client/package.json [hunk 1]
```diff
@@ -5,16 +5,16 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
+    "start": "node .next/standalone/server.js",
```
Comment (abhijeetnishal) [inline client/package.json:8]: Never change system project settings

File: client/src/components/ThemeToggle.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+// // components/ThemeToggle.tsx
+// "use client";
+// import { useTheme } from "next-themes";
+// import { useEffect, useState } from "react";
```
Comment (abhijeetnishal) [inline client/src/components/ThemeToggle.tsx:4]: Try to remove commented code

**Review by abhijeetnishal — APPROVED** [review]
I have approved this PR but i have added comments don't repeat this for other projects.

File: client/package.json (hunk 1 above)
Comment (Prashant-2024) [reply client/package.json:8]: Got it.

File: client/src/components/ThemeToggle.tsx (hunk 2 above)
Comment (Prashant-2024) [reply client/src/components/ThemeToggle.tsx:4]: Removed

**Pull request merged**